cylc/uiserver/main.py

```diff
@@ -119,6 +120,10 @@ def _make_app(self, debug: bool):
                     UIServerGraphQLHandler,
```

Comment:
Was GraphiQL was using the it's own handler in our example?
https://github.com/kinow/tornado-sandbox/blob/f1b6e75be2081b12025c045d26350801ab65a511/web4/app.py#L14-L16

And I think this can be just `/graphiql` now (as I believe the JS uses both endpoints)

Reply:
It was. This way it would serve the custom template for the graphiql endpoint. And this custom template has the React app to consume websockets (which was re-rendering the query whenever a websocket message arrived... in our case a date/time and a fruit I think haha).

I wasn't going to add this to avoid bloating cylc-uiserver unnecessarily (we will have cylc-ui to test it) but maybe it's a good idea, so that we can test the subscriptions without the cylc-ui as well?

Reply:
> I wasn't going to add this to avoid bloating cylc-uiserver unnecessarily (we will have cylc-ui to test it) but maybe it's a good idea, so that we can test the subscriptions without the cylc-ui as well?

Maybe we could migrate it to Apollo? (or maybe there's one out there already?)

Reply:
Yeah, for now keeping it to minimum changes required, so simply copied the template from the PR for now. Leaving the change of `/graphql/graphiql` to `/graphiql` for later too :+1: